src/sentry/incidents/utils/types.py

```diff
@@ -13,3 +13,7 @@ class QuerySubscriptionUpdate(TypedDict):
 class AlertRuleActivationConditionType(Enum):
     RELEASE_CREATION = 0
     DEPLOY_CREATION = 1
+
+
+class DataSourceType(StrEnum):
+    SNUBA_QUERY_SUBSCRIPTION = "snuba_query_subscription"
```

Comment:
nit: can probably just be a constant:
```suggestion
DATA_SOURCE_SNUBA_SUBSCRIPTION = "snuba_query_subscription"
```
(or something like that) -- as long as we only have the 1 data source in `incidents` module.